aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1012,22 +1010,21 @@ struct HelperInterpBase {
 
     // Rescale float values to int16
     int16_t * data_i16 = (int16_t *) data_f64;
-    auto aligned_interp_size = interp_size;
 
     if (align_i32) {
       // We should respect int32 alignment as we will load int16 data as int32
       // See ImagingResampleHorizontalConvolution8u4x, mmk0 = _mm256_set1_epi32(*(int32_t*)&k[x]);
-      // compute aligned_interp_size = nearest pair value to interp_size
+      // compute aligned_interp_size = nearest pair value to aligned_interp_size
       while (aligned_interp_size % sizeof(int32_t) != 0) {
         aligned_interp_size += 1;
       }
       // assert that we wont go out of bounds
-      TORCH_INTERNAL_ASSERT(aligned_interp_size * sizeof(int16_t) < interp_size * sizeof(double));
+      TORCH_INTERNAL_ASSERT(aligned_interp_size * sizeof(int16_t) < aligned_interp_size * sizeof(double));
     }
 
     for (const auto j : c10::irange(output_size)) {
-      for (const auto k : c10::irange(interp_size)) {
-        double v = data_f64[j * interp_size + k] * (1 << weights_precision);
+      for (const auto k : c10::irange(aligned_interp_size )) {
+        double v = data_f64[j * aligned_interp_size + k] * (1 << weights_precision);
```

Comment:
Note sure if this is the issue causing CI to fail, but this should probably stay as interp_size, and in line 997 we should keep it as interp_size. And undelete 1015.